MockKit/MockCGMManager.swift

```diff
@@ -395,6 +434,18 @@ extension MockCGMState: RawRepresentable {
             rawValue["localizedMessage"] = cgmStatusHighlight.localizedMessage
             rawValue["alertIdentifier"] = cgmStatusHighlight.alertIdentifier
         }
+        
+        if let cgmStatusProgress = cgmStatusProgress {
+            rawValue["progressPercentComplete"] = cgmStatusProgress.percentComplete
+        
+            if let warningThresholdPercentValue = cgmStatusProgress.warningThresholdPercentValue {
+                rawValue["warningThresholdPercentValue"] = warningThresholdPercentValue
```

Comment:
You might want to prefix `"warningThresholdPercentValue"` and `"criticalThresholdPercentValue"` with `progress` so it is clear that all of these belong to the Progress. So, `"progressPercentComplete"`, `"progressWarningThresholdPercentValue"`, and `"progressCriticalThresholdPercentValue"`.